packages/analytics-react-native/android/src/main/AndroidManifestNew.xml

```diff
@@ -0,0 +1,3 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
```

Comment:
Can we remove this permission? Concerned it will block app approvals.

Maintenance - /android/src/main/AndroidManifest.xml

Reply:
The file is a copy of existing https://github.com/amplitude/Amplitude-TypeScript/blob/main/packages/analytics-react-native/android/src/main/AndroidManifest.xml
Remove in both files?

Reply:
Amplitude-ReactNative didn't have this permission.

Reply:
I think we determined to use an example plugins to fetch the location? @justin-fiedler

Reply:
Thanks I updated this ticket with proposed changes. PTAL - https://amplitude.atlassian.net/browse/AMP-62860?focusedCommentId=204509

Reply:
Without the permission there is a lint error in a sample application - let's try to fix the issue in the AMP-62860 ticket:
```
Error: Missing permissions required by LocationManager.getLastKnownLocation: android.permission.ACCESS_COARSE_LOCATION or android.permission.ACCESS_FINE_LOCATION [MissingPermission]
          location = locationManager.getLastKnownLocation(provider!!)
                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

   Explanation for issues of type "MissingPermission":
   This check scans through your code and libraries and looks at the APIs
   being used, and checks this against the set of permissions required to
   access those APIs. If the code using those APIs is called at runtime, then
   the program will crash.

   Furthermore, for permissions that are revocable (with targetSdkVersion 23),
   client code must also be prepared to handle the calls throwing an exception
   if the user rejects the request for permission at runtime.
```